scripts/db_clean.ts

```diff
@@ -8,6 +8,7 @@ type MutationName = keyof ValueTypes['mutation_root'];
     'delete_vault_transactions',
     'delete_pending_vault_transactions',
     'delete_claims',
+    'delete_contributions',
```

Comment:
this is already declared once above

Reply:
@zashton I ran into a fk error on deletion, so I guess we need to move it up